coralproject/talk
# Lock install issue

## What does this PR do?
Fix wrong syntax on `serve.js` file. 
If you set `TALK_INSTALL_LOCK=TRUE` on environment and try to start, system you throw an error `'install lock active'` because the switch statement has a syntax error.

## How do I test this PR?
- Set `TALK_INSTALL_LOCK=TRUE`
- Try to start application `sudo yarn start`
- See the application running
